packages/algorithm/lib/variants/bgoncharuck/base_hardcoded.dart

```diff
@@ -0,0 +1,13 @@
+part of 'algorithm.dart';
+
+class _Hardcoded implements _OldWisdom {
+  Map<ProphecyId, ProphecyEntity> says(UserEntity aboutUser, int inTimeOf) => {
+        ProphecyId.INTERNAL_STRENGTH:
+            ProphecyEntity(id: ProphecyId.INTERNAL_STRENGTH, value: 49),
+        ProphecyId.MOODLET: ProphecyEntity(id: ProphecyId.MOODLET, value: 65),
+        ProphecyId.AMBITION: ProphecyEntity(id: ProphecyId.AMBITION, value: 90),
+        ProphecyId.INTELLIGENCE:
+            ProphecyEntity(id: ProphecyId.INTELLIGENCE, value: 53),
+        ProphecyId.LUCK: ProphecyEntity(id: ProphecyId.LUCK, value: 70),
+      };
+}
```

Comment:
`_Hardcoded ` does not seem to be used anywhere , shall it be removed?

Reply:
It is used in https://github.com/bgoncharuck/my_prophet/blob/main-screen-without-top-bar/packages/algorithm/lib/variants/bgoncharuck/algorithm.dart#L23